metabase/metabase
# Reset temporal unit parameter value if it becomes no longer valid after temporal unit list changes

Epic https://github.com/metabase/metabase/issues/42118
Demo https://www.loom.com/share/c736e1ffa6a14565b990b9ed295457c5

How to verify:
- Add a question with a breakout based on a date column to a dashboard
- Add a temporal unit parameter and map it to the breakout column
- Set the parameter value e.g. to "Year"
- Modify the list of allowed temporal units for this parameter to exclude "Year"
- The value of the parameter should be reset
- As a side note, the parameter value will be reset when toggling All/None in the temporal unit picker
